2_adventures/001/kotp/04_automated_testing_example/spec/fizz_buzz_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'spec_helper'
+require 'fizz_buzz'
+
+describe FizzBuzz do
+  describe 'Single Numbers' do
+    it 'must return 1 for 1' do
```

Comment:
When I started writing specs (with Rspec) I followed a great series ([part 1](http://everydayrails.com/2012/03/12/testing-series-intro.html))by Aaron Summer of [Everyday Rails](http://everydayrails.com). I learned that you should avoid duplication by not writing `it 'should'`or variations like _must_ in this case. With Rspec's documentation formatter the output becomes nicely readable:

```sh
rspec spec/models/course_spec.rb

Course
  has a factory
  has a title
  has a description
  [snip]

Finished in 0.39821 seconds (files took 0.63053 seconds to load)
11 examples, 0 failures
```

My feedback would be to get rid of the must and to just write `it 'returns 1'`. In my opinion you could also skip the input value because it's explicitly used in the method call in the next line.

I found [Better Specs](http://betterspecs.org) a nice collection of helpful tips for writing specs.